doc/sphinx/history-and-changes.rst

```diff
@@ -0,0 +1,18 @@
+==========================
+History and recent changes
+==========================
+
+This chapter is divided in two sub-chapters.  The first one is about
```

Comment:
I used the word "sub-chapter" to designate what used to be chapters and is still on separate pages. But I agree that this terminology is not fit to be left over in the reference manual.

Reply:
I'll keep it like this for the time being though, because most of these sub-chapters are still designated as chapters wherever they are linked to. We can revise this once the refactoring has done some progress.

Reply:
... has made some progress.

Reply:
Sounds good